File: okex-cs-sdk-api/OKExSDKLibrary/Models/Account/Wallet.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace OKExSDK.Models.Account
{
    public class Wallet
    {
        /// <summary>
        /// 币种，如btc
        /// </summary>
        public string currency { get; set; }
        /// <summary>
        /// 余额
        /// </summary>
        public decimal balance { get; set; }
        /// <summary>
        /// 冻结(不可用)
        /// </summary>
        public decimal hold { get; set; }
        /// <summary>
        /// 可用于提现或资金划转的数量
        /// </summary>
        public decimal available { get; set; }
    }
}

File: okex-java-sdk-api/src/main/java/com/okcoin/commons/okex/open/api/bean/account/result/Wallet.java
package com.okcoin.commons.okex.open.api.bean.account.result;

public class Wallet {

    private String currency;
    private String balance;
    private String hold;
    private String available;

    public String getCurrency() {
        return currency;
    }

    public void setCurrency(String currency) {
        this.currency = currency;
    }

    public String getBalance() {
        return balance;
    }

    public void setBalance(String balance) {
        this.balance = balance;
    }


    public String getHold() {
        return hold;
    }

    public void setHold(String hold) {
        this.hold = hold;
    }

    public String getAvailable() {
        return available;
    }

    public void setAvailable(String available) {
        this.available = available;
    }
}
